chsdi/templates/htmlpopup/cadastralwebmap.mako

```diff
@@ -9,25 +9,30 @@
   from gatilegrid import getTileGrid
   from chsdi.models.vector import get_scale
   from chsdi.lib.validation.identify import IdentifyServiceValidation
+  from chsdi.lib.helpers import shift_to
   request = context.get('request')
-  grid = getTileGrid(21781)()
-  defaultExtent = ','.join(map(str, grid.extent))
-  defaultImageDisplay = '400,600,96'
   fallbackLang = 'fr' if request.lang in ('fr', 'it') else 'de'
 
   class CadastralWebMapParams(IdentifyServiceValidation):
       def __init__(self, request):
+          self.srid = request.params.get('sr', '21781')
+          grid = getTileGrid(self.srid)()
+          defaultImageDisplay = '400,600,96'
+          defaultExtent = ','.join(map(str, grid.extent))
+
           self.mapExtent = request.params.get('mapExtent', defaultExtent)
           self.imageDisplay = request.params.get('imageDisplay', defaultImageDisplay)
-  params = CadastralWebMapParams(request)
+          self.coord = request.params.get('coord')
 
-  c['bbox'] = params.mapExtent.bounds
-  c['bboxlv95'] = [c['bbox'][0] + 2000000, c['bbox'][1] + 1000000,
-                   c['bbox'][2] + 2000000, c['bbox'][3] + 1000000]
+  params = CadastralWebMapParams(request)
+  c['bboxlv95'] =  list(params.mapExtent.bounds)
```

Comment:
if sr parameter == 2056 then this is incorrect...
why did you remove the if we added yesterday?

Reply:
21781 I mean

Reply:
and you need the fix in geoadmin otherwise it won't work.

Reply:
Yes I removed it because I noticed that the params coord and mapExtent are delivered per default in Lv95 also when sr=21781.

Reply:
no, incorrect default is lv03 still, you have to pass in sr parameter....
If you tested with prod it's normal it didn't work.